lms/djangoapps/dashboard/tasks.py

```diff
@@ -0,0 +1,100 @@
+"""
+This file contains celery tasks sysdashboard
+"""
+import logging
+import time
+import StringIO
+import xlwt
+
+from celery import task
+from django.core.cache import cache
+from django.conf import settings
+from django.core.mail import EmailMessage, get_connection
+from django.utils.translation import ugettext as _
+
+from student.roles import CourseStaffRole, CourseInstructorRole
+from xmodule.modulestore.django import modulestore
+from .models import EmailsAddressMailing
+
+
+@task(name='mass_sending_report')
+def mass_sending_report():
+    connection = get_connection()
+    xls_file = get_courses_xls()
+    email_list = []
+    for email in EmailsAddressMailing.objects.all():
+        if email.active:
+            email_message = EmailMessage('Catalogue Email Service', 'Hi all,\nThe list of courses is in the attachment.', settings.EMAIL_FROM, [email.email],)
+            email_message.attach('Courses_report.xls', xls_file.getvalue(), 'application/vnd.ms-excel')
+            email_list.append(email_message)
+    connection.send_messages(tuple(email_list))
+    
+@task()
+def send_report_email(request, email_adress):
+    xls_file = get_courses_xls()
+    email_message = EmailMessage('Catalogue Email Service', 'Hi all,\nThe list of courses is in the attachment.', settings.EMAIL_FROM, [email_adress],)
+    email_message.attach('Courses_{0}.xls'.format(request.META['SERVER_NAME']), xls_file.getvalue(), 'application/vnd.ms-excel')
+    email_message.send()
+
+@task()
+def get_courses_xls():
+    """
+    Getting course data and return xls file
+    _____________________________________________________________________________________
+    | Course URL | Course Run ID  | Course name | Enrollment end date | Course end date |
+    """
+    data = []
+    roles = [CourseInstructorRole, CourseStaffRole, ]
+    def_ms = modulestore()
```

Comment:
What does `def_ms` mean? Defined modulestore? Please describe or change the name to something understandable.